pkg/sloop/webserver/webfiles/index.html

```diff
@@ -75,7 +75,15 @@
 
             <input type="submit">
         </form>
-
+        <!-- Toggle switch to show payload changes -->
+        <div class="switchdiv">
```

Comment:
```suggestion
        <div class="payload-changes-switch-container">
```

Reply:
**Suggestion**: Add classes based on the need
**Example**: switchpara can be accessed `.payload-changes-switch-container > p`

Reply:
I edited it to `switch-container` and `.switch-container p`. If we create other toggles in the future - we can use the same class name.